source/gui/chartView/ChartTimeRangeBar.cpp

```diff
@@ -19,18 +24,18 @@ ChartTimeRangeBar::~ChartTimeRangeBar() {
 
 void
 ChartTimeRangeBar::InitTabs() {
-    BButton *oneDayButton = createButton("1 D", new BMessage(TimeRange::M_DAY));
+    BButton *oneDayButton = createButton(B_TRANSLATE("1 D"), new BMessage(TimeRange::M_DAY));
```

Comment:
When using abbreviations, or any non-obvious strings in the GUI, it's a good idea to use B_TRANSLATE_COMMENT to help the translators.

```suggestion
    BButton *oneDayButton = createButton(B_TRANSLATE_COMMENT("1 D", "Abbreviation for 'one day', short as possible."),
new BMessage(TimeRange::M_DAY));
```